intellisense not work

## Environment data
SDK de .NET Core (reflejando cualquier global.json):
 Version:   2.1.301
 Commit:    59524873d6

Entorno de tiempo de ejecución:
 OS Name:     Windows
 OS Version:  10.0.17134
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\2.1.301\

Host (useful for support):
  Version: 2.1.1
  Commit:  6985b9f684

VS Code version: 
1.25.0

C# Extension version:
1.15.2

the intellisense not work in code web api
when i try to restar onmisharp see "command 'o.restart' not found"

i'm having the same issue.
No output for c# or omnisharp.
I opened the developer tools and found this:

`[Extension Host] Activating extension 'ms-vscode.csharp' failed: Cannot find module 'rxjs/Subscription'.`

--- a/src/omnisharp/server.ts
+++ b/src/omnisharp/server.ts
@@ -403,11 +403,11 @@ export class OmniSharpServer {
         }
 
         if (options.sdkPath.length > 0) {
-            args.push(`Sdk:Path='${options.sdkPath}'`);
+            args.push(`Sdk:Path=${options.sdkPath}`);
         }
 
         if (options.sdkVersion.length > 0) {
-            args.push(`Sdk:Version='${options.sdkVersion}'`);
+            args.push(`Sdk:Version=${options.sdkVersion}`);
         }
 
         if (options.sdkIncludePrereleases) {

